galley/cmd/galley/cmd/probe.go

```diff
@@ -32,13 +32,13 @@ func probeCmd() *cobra.Command {
 		Short: "Check the liveness or readiness of a locally-running server",
 		Run: func(cmd *cobra.Command, _ []string) {
 			if !probeOptions.IsValid() {
-				fmt.Fprintf(cmd.OutOrStdout(), "some options are not valid")
+				_, _ = fmt.Fprintf(cmd.OutOrStdout(), "some options are not valid")
```

Comment:
My Go noobness may be showing, but can you explain what this is for?

Reply:
The _, _ is for explicitly ignoring the return values of fmt.Fprintf.